lib/elixir_internal_certificate/scraper/scrapers.ex

```diff
@@ -4,18 +4,34 @@ defmodule ElixirInternalCertificate.Scraper.Scrapers do
   """
 
   alias ElixirInternalCertificate.Repo
-  alias ElixirInternalCertificate.Scraper.Schemas.UserSearch
+  alias ElixirInternalCertificate.Scraper.Schemas.{SearchResult, UrlResult, UserSearch}
 
   def insert_search_keywords(attrs),
-    do: Repo.insert_all(UserSearch, attrs)
+    do: Repo.insert_all(UserSearch, attrs, returning: true)
 
-  def create_search_keyword(keywords, user) do
+  def create_user_search(keywords, user) do
     keywords
     |> parsing_keywords(user)
     |> insert_search_keywords()
-    |> elem(0)
   end
 
+  def get_user_search(id), do: Repo.get(UserSearch, id)
+
+  def update_user_search_status(user_search, status) do
+    user_search
+    |> UserSearch.status_changeset(status)
+    |> Repo.update()
+  end
+
+  def saving_search_result(result) do
+    %SearchResult{}
+    |> SearchResult.search_result_changeset(result)
+    |> Repo.insert(returning: true)
+  end
+
+  def saving_url(result),
```

Comment:
should we just name the method starting with a verb -> save_url?

also seem we are using `Repo.insert_all` here, the result here isn't clear it's a single item or multiple items, how about tweak it a bit like `create_url_results(url_results)`